From 3be7e9a806942594c2d18af799994c6534f8e2fd Mon Sep 17 00:00:00 2001
From: Lokesh Kumar Goel <lokeshkumar.goel@lge.com>
Date: Wed, 26 Sep 2018 14:33:42 -0700
Subject: [PATCH] Revert "Remove service permissions related to wam"

:Release Notes:
This reverts commit ae4a2ee08fc56851a1e680849799cf67c1e8b4ac.

:Detailed Notes:
wam with chromium68 is still not ready for ACG
Remove this when wam with new chromium is ready

:Testing Performed:

:QA Notes:

:Issues Addressed:
---
 files/sysbus/compat.perm.json | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/files/sysbus/compat.perm.json b/files/sysbus/compat.perm.json
index 638011c..e4215ed 100755
--- a/files/sysbus/compat.perm.json
+++ b/files/sysbus/compat.perm.json
@@ -4,6 +4,8 @@
   "com.palm.power": ["private", "public"],
   "com.palm.sleep": ["private", "public"],
   "com.palm.systemmanager": ["private", "public"],
+  "com.palm.webappmanager": ["private"],
+  "com.palm.webappmgr": ["private", "public"],
   "com.webos.applicationManager": ["private", "public"],
   "com.webos.booster": ["private"],
   "com.webos.media": ["private", "public"],
-- 
1.9.1

